spec/ParseObject.spec.js

```diff
@@ -568,6 +568,21 @@ describe('Parse.Object testing', () => {
       );
   });
 
+  it_only_db('mongo')('can increment array nested fields', async () => {
+    const obj = new TestObject();
+    obj.set('items', [ { value: 'a', count: 5 }, { value: 'b', count: 1 } ]);
+    await obj.save();
+
+    obj.increment('items.0.count', 15);
```

Comment:
Is this notation MongoDB specific? If so, this may not be clear for developers who use Postgres. Should this notation be documented?

Reply:
I got dot notation working on PG years ago. Adding Array support might be possible.

Reply:
It's not a requirement for this PR to add support for PG, I was just referring to the syntax itself, whether that is commonly known or should be documented.